comments.php

```diff
@@ -36,13 +36,13 @@
 			<?php wp_list_comments( array( 'callback' => 'twentytwelve_comment', 'style' => 'ol' ) ); ?>
 		</ol><!-- .commentlist -->
 
-		<?php if ( get_comment_pages_count() > 1 && get_option( 'page_comments' ) ) : // are there comments to navigate through ?>
+		<?php if ( get_comment_pages_count() > 1 && get_option( 'page_comments' ) ) : // Are there comments to navigate through? ?>
```

Comment:
One of the community standards calls for inline comments to be proper sentences - so they need to end on punctuation. Given the nature of the comment, a question mark seemed appropriate - but it does look weird to see a question mark to end the comment, followed by the `?>` PHP closure.